app/javascript/app/styles/settings.scss

```diff
@@ -57,14 +57,15 @@ $midnight-blue: #0a1941;
 $blue-gradient: linear-gradient(282deg, $light-blue, $dark-blue);
 $elephant: #113750;
 $athens-gray: #e6e7ed;
+$link-water: #ecf1fa;
```

Comment:
I talked with Fanny and we should make a task to use just the colors in the UI kit. Maybe this is one of them. Just making a mental note to myself 😝